common/middleware/logger/geoip.go

```diff
@@ -0,0 +1,36 @@
+package logger
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/MurmurationsNetwork/MurmurationsServices/common/httputil"
+	"github.com/gin-gonic/gin"
+)
+
+type respond struct {
+	Data geoInfo `json:"data,omitempty"`
+}
+
+type geoInfo struct {
+	City    string  `json:"city,omitempty"`
+	Country string  `json:"country,omitempty"`
+	Lat     float64 `json:"lat,omitempty"`
+	Lon     float64 `json:"lon,omitempty"`
+}
+
+func getGeoInfo(c *gin.Context) *geoInfo {
+	bytes, err := httputil.GetByte(fmt.Sprintf("http://geoip-app:8080/city/%s", c.ClientIP()))
```

Comment:
If we run locally, `c.Client IP()` will return the internal IP address.

To test it, please change `c.ClientIP()` to a real IP and you will get the correct information.

```
bytes, err := httputil.GetByte(fmt.Sprintf("http://geoip-app:8080/city/%s", "1.200.15.45"))
```

![Screen Shot 2021-04-28 at 5 52 56 PM](https://user-images.githubusercontent.com/11765228/116384595-98d3dd80-a84a-11eb-87e4-3ac68c27773a.png)